jujugui/static/gui/src/app/app.js

```diff
@@ -1774,7 +1784,13 @@ YUI.add('juju-gui', function(Y) {
           this._renderMachineView.bind(this),
           this._clearMachineView.bind(this)],
         ['gui.inspector',
-          this._renderInspector.bind(this)]
+          this._renderInspector.bind(this)],
+        ['gui.deploy',
+          this._renderDeployment.bind(this),
+          this._clearDeployment.bind(this)],
+        ['deploy',
```

Comment:
I have no idea why, but I was getting messages for both "No dispatcher found for key: deploy" and "No dispatcher found for key: gui.deploy".